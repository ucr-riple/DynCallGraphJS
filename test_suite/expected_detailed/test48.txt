{
    'global (test48.js:8:15:8:28)': [
        'makeCounter (test48.js:1:1:6:2)'
    ],
    'global (test48.js:9:1:9:10)': [
        'anon (test48.js:3:10:5:4)'
    ],
    'global (test48.js:10:1:10:10)': [
        'anon (test48.js:3:10:5:4)'
    ]
}