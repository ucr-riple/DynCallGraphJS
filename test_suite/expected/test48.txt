{ '(test48.js:8:15:8:28)':
   [ '(test48.js:1:1:6:2)' ],
  '(test48.js:9:1:9:10)':
   [ '(test48.js:3:10:5:4)' ],
  '(test48.js:10:1:10:10)':
   [ '(test48.js:3:10:5:4)' ] }